### DISCUSSION TITLE
SMS Android Check

### DISCUSSION THREAD

**Discussion by atvnuk** [mvt-project/mvt#263] [Q&A]
Hi

I noticed that the incorrect number of SMS with links are detected for MVT-Android. How can I ensure that the right amount of SMS with links are being detected?

**Comment by Te-k** [comment]
Hi, without more information, I can't really debug what is happening. It is possible that the current regular expression used to catch links isn't working. Do you have examples of SMS that are not detected by MVT-Android?
